src/timeUtils.ts

```diff
@@ -1086,21 +1109,24 @@ export function getTimeZoneName() {
 }
 
 /**
- * Removes milliseconds from a date string if it's in DOY (Day of Year) or ISO 8601 time format.
+ * Removes milliseconds from a date string if it's in ISO Ordinal or ISO 8601 (UTC) time format.
  * If the string is not a recognized time format, it returns the original string unchanged.
  *
  * @param {string} dateString - The date string to process.
  * @returns {string} The date string with milliseconds removed, or the original string if no milliseconds were present or the format wasn't recognized.
  *
  * @example
- * removeDateStringMilliseconds('2023-365T12:30:45.123'); // returns '2023-365T12:30:45'
- * removeDateStringMilliseconds('2023-12-25T08:00:00.500Z'); // returns '2023-12-25T08:00:00Z'
- * removeDateStringMilliseconds('invalid-string'); // returns 'invalid-string'
+ * removeUtcOrIsoStringMilliseconds('2023-365T12:30:45.123'); // returns '2023-365T12:30:45'
+ * removeUtcOrIsoStringMilliseconds('2023-12-25T08:00:00.500Z'); // returns '2023-12-25T08:00:00Z'
+ * removeUtcOrIsoStringMilliseconds('invalid-string'); // returns 'invalid-string'
  *
  */
-export function removeDateStringMilliseconds(dateString: string): string {
-  if (validateTime(dateString, TimeTypes.ABSOLUTE)) {
-    return dateString.split('.')[0];
+export function removeUtcOrIsoStringMilliseconds(utcOrIsoString: string): string {
```

Comment:
Any reason not to capitalize UTC in this and other function names?

Reply:
Using 'Utc' instead of 'UTC' improves readability for me. For me, 'UtcOrIso' is easier to visually parse than 'UTCOrIso'. When fully capitalized, my brain tends to read it as 'UTCO-r-Iso', which is not the intended meaning.

Reply:
Fair enough!